src/FlashMessage/FlashManager.php

```diff
@@ -18,8 +18,10 @@
 
 /**
  * @author Vincent Composieux <composieux@ekino.com>
+ *
+ * NEXT_MAJOR: remove StatusClassRendererInterface
```

Comment:
This comment would be better on the StatusClassRendererInterface file

Reply:
We dont wanna remove StatusClassRendererInterface. I correct it. Should be remove StatusClassRendererInterface implementation